### ISSUE TITLE
notice / notification logging from PostgreSQL is not working

### ISSUE THREAD

**Issue by gv-beemining** [sequelize/sequelize#11532]
## Issue Description

Im trying to use afterConnect hook to intercept notice and notiification events from a PostgreSQL database

### What are you doing?

```js
const config = {
  username: 'username',
  password: 'password',
  database: 'database',
  host: 'host',
  dialect: 'postgres',
  logging: true,
  pool: {
    max: 1,
    min: 1,
    acquire: 30000,
    idle: 10000
  }
}

const sequelize = new Sequelize(config);

sequelize.addHook('afterConnect', (client) => {
  //never fired
  client.on('notice', msg => {
    console.log('notice:', msg)
  })
  //never fired
  client.on('notification', msg => {
    console.log(msg.channel)
    console.log(msg.payload)
  })
});

await sequelize.query('any query that triggers a notice or notification');
```

### What do you expect to happen?

I wanted to the notice or notification event called at least once

### What is actually happening?

The events are never fired

```
Output here
```

### Environment

- Sequelize version: sequelize@5.19.4
- Node.js version: v9.11.1
- Operating System: Ubuntu 16.04

### How does this problem relate to dialects?

- [ ] I think this problem happens regardless of the dialect.
- [ X] I think this problem happens only for the following dialect(s): pg@7.12.1
- [ ] I don't know, I was using PUT-YOUR-DIALECT-HERE, with connector library version XXX and database version XXX

### Would you be willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time and I know how to start.
- [X ] Yes, I have the time but I don't know how to start, I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

**Comment by papb** [comment]
You have posted a code snippet that shows the problem but is not entirely self-contained (i.e. I can't just copy-paste it and run it). Please provide a [SSCCE](http://sscce.org/) (also known as [MCVE/reprex](https://stackoverflow.com/help/minimal-reproducible-example)) to allow contributors to work on this on a timely manner. I suggest using the [sequelize-sscce repository](https://github.com/papb/sequelize-sscce) for this.

**Comment by gv-beemining** [comment]
SSCCE created but how i can 'simulate' a query that triggers a raise notice to test the issue?

Thanks!

**Comment by papb** [comment]
@gv-beemining Well, I return the question to you... Don't you have an example of query in mind? Just put that in your `sequelize.query` call... Or perhaps I didn't understand your question.

**Comment by gv-beemining** [comment]
repo query updated, tests passed, Thanks!

**Comment by papb** [comment]
@gv-beemining Looking at [it](https://github.com/papb/sequelize-sscce/pull/10)'s [log output](https://travis-ci.com/papb/sequelize-sscce/jobs/244307106#L519), looks like indeed your `console.log`s didn't run. Are you sure they would work with the connector library directly (i.e. without sequelize)?

**Comment by gv-beemining** [comment]
> @gv-beemining Looking at [it](https://github.com/papb/sequelize-sscce/pull/10)'s [log output](https://travis-ci.com/papb/sequelize-sscce/jobs/244307106#L519), looks like indeed your `console.log`s didn't run. Are you sure they would work with the connector library directly (i.e. without sequelize)?

yes, i tried without sequelize, the library in this case is pg (example code):

``

 const client = new `Client()
    client.on('notice', msg => console.warn('notice:', msg))
    client.on('notification', msg => {
      console.log(msg.channel) // 
      console.log(msg.payload) // 
    })

    await client.connect();
    await client.query("DO language plpgsql $$ BEGIN RAISE NOTICE 'hello, world!'; END $$;");

**Comment by h0gar** [comment]
Scratched my head over this for an hour.

By default sequelize only sends message with a level at or above "warning": https://sequelize.org/docs/v6/other-topics/dialect-specific-things/#postgresql
